xml/System.Security.Cryptography/HMAC.xml

```diff
@@ -611,11 +620,22 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="destination">The buffer to receive the HMAC value.</param>
+        <param name="bytesWritten">When this method returns, the total number of bytes written into <paramref name="destination"/>. This parameter is treated as uninitialized.</param>
+        <summary>Attempts to finalize the HMAC computation after the last data is processed by the HMAC algorithm.</summary>
+        <returns><see langword="true"/> if <paramref name="destination"/> is long enough to receive the HMAC value; otherwise, <see langword="false"/>.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+The default implementation of this method is to throw a <xref:System.PlatformNotSupportedException>,
+because there is no managed implementation of HMAC.
+
+Derived types must override this method.
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.PlatformNotSupportedException">The method is not overriden in the derived type.</exception>
```

Comment:
```suggestion
        <exception cref="T:System.PlatformNotSupportedException">In all cases.</exception>
```

Reply:
I don't think this suggestion is accurate. The method will not throw if the derived class implements it.